pagopa/pn-frontend
# Rework special contacts - pn-12055

## Short description
Reworked special contacts components

## List of changes proposed in this pull request
- Updated SpecialContacts.tsx
- Updated SpecialContactElem.tsx
- Merged types LegalChannelType and CourtesyChannelType into ChannelType
- Updated tests

## How to test
Login as pf/pg -> go to recapiti page -> add some special address and check that everything is ok
